### PR TITLE
Events scroll

### PR DESCRIPTION
**Pull request by dlangston**
@DanielJMaher I still need to fixe some css on this but it is working the way we want.

### CODE REVIEW

**Comment by birdage** [comment]
@dlangston im going to hold on this till we chat

File: ooiui/static/css/common/plotting.css [hunk 1]
```diff
@@ -1,175 +1,257 @@
...
+    width: 15%;
+    float: left;
+}
+.header-fixed > thead > tr > th {
+    border: 1px solid #ddd;
+  
+}
+
+     
+  /* table { */
```
Comment (birdage) [inline ooiui/static/css/common/plotting.css]: large comment section should be removed

**Comment by DanielJMaher** [comment]
This is only here so i could pull it down if I needed to.. He is not done yet.

**Comment by birdage** [comment]
@dlangston how goes it?

**Comment by dlangston** [comment]
Dan pulled me off of it to fix a bug with the search box on the streams page. I am in the process of trying to get the table header to size correctly with the body, but having trouble with overlapping.

**Comment by birdage** [comment]
ok

**Comment by birdage** [comment]
@dlangston done?

**Comment by dlangston** [comment]
Almost, but Dan pulled me off to work on the hovers on the science map page.

**Comment by dlangston** [comment]
Had to create a new PR of this one after rebasing

**Pull request closed without merging**